src/test/regress/expected/nested_execution.out

```diff
@@ -0,0 +1,138 @@
+SET search_path TO nested_execution;
+SET citus.enable_local_execution TO on;
```

Comment:
It seems useful to allow distributed queries from queries on shards, queries on Citus local tables, and queries on local intermediate results, since the coordinator can coordinate in those cases. That does cause different behaviour if enable_local_execution is disabled.